### ISSUE TITLE
[bug]: Tt.auth.getAllowedAuthProviders is not a function.

### ISSUE THREAD

**Issue by jayeshanandani** [hoppscotch/hoppscotch#4261]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

After generating desktop build for MAC, it shows following error when we click on login button.

TypeError: Tt.auth.getAllowedAuthProviders is not a function. (In 'Tt.auth.getAllowedAuthProviders()', 'Tt.auth.getAllowedAuthProviders' is undefined)

![image](https://github.com/user-attachments/assets/1a92d0ff-645b-4be5-9590-61794fe39bc8)

### Steps to reproduce

1. Generated build of desktop using `npm run tauri build`.
2. Click on Login.

### Environment

Release

### Version

Self-hosted

**Comment by shipko** [comment]
Yes, it's a known bug. You need to implement your own method in desktop platform file (auth.ts)

**Comment by jayeshanandani** [comment]
thanks @shipko, have added code.
![Uploading Screenshot 2024-08-28 at 1.03.58 PM.png…]()

**Comment by liyasthomas** [comment]
Thanks for your patience and valuable feedback. We’ve fixed this bug in the latest release.
[Download the latest builds](https://hoppscotch.com/download) and provide your valuable feedback.

Closing this ticket as this bug is now fixed in the recent version. Please feel free to reach out if you have any other concerns.
